README.md

```diff
@@ -9,3 +9,16 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+## Answer 
+A bijection is established when each node in $G_{1}$ uniquely maps to a comparable node in $G_{2}$. Given that $G_{1}$ and $G_{2}$ contain the same number of nodes, we can define a mapping function _f_ that creates this one-to-one correspondence between them.
```

Comment:
Can you prove that such a mapping function exists?

Reply:
I added the one-to-one and onto proof.